assets/src/data/model/datetime/index.js

```diff
@@ -89,22 +91,24 @@ export const whereConditions = ( {
 	month = 'none',
 } ) => {
 	const where = [];
-	const GREATER_AND_EQUAL = encodeURIComponent( '>=' );
-	const LESS_AND_EQUAL = encodeURIComponent( '<=' );
-
 	if ( ! showExpired ) {
-		where.push( 'where[DTT_EVT_end**expired][]=>&where[DTT_EVT_end**expired][]=' +
-			nowDateAndTime.local().format() );
+		where.push(
+			'where[DTT_EVT_end**expired][]=' + GREATER_THAN +
+			'&where[DTT_EVT_end**expired][]=' +
+			nowDateAndTime.local().format()
+		);
 	}
 	if ( month && month !== 'none' ) {
-		where.push( 'where[DTT_EVT_start][]=' +
-			GREATER_AND_EQUAL +
+		where.push(
+			'where[DTT_EVT_start][]=' + GREATER_THAN +
```

Comment:
Why did you change this from `GREATER_AND_EQUAL` to `GREATER_THAN`?  I think it should remain as it was as we're wanting to _include_ the start of the month right?